clients/venice-push-job/src/main/java/com/linkedin/venice/heartbeat/DefaultPushJobHeartbeatSender.java

```diff
@@ -77,7 +77,7 @@ class DefaultPushJobHeartbeatSender implements PushJobHeartbeatSender {
   }
 
   // For test purpose.
-  VeniceWriter<byte[], byte[], byte[]> getVeniceWriter() {
+  protected VeniceWriter<byte[], byte[], byte[]> getVeniceWriter() {
```

Comment:
This change is not necessary, from my understanding.

Reply:
fixed.